src/SmartComponents/HybridInventoryTabs/helpers.js

```diff
@@ -87,3 +91,42 @@ export const useActionResolver = (handleModalToggle) =>
     ],
     []
   );
+
+export const useOnLoad = (filters) => {
+  const store = useStore();
+  return useCallback(
+    ({ mergeWithEntities, INVENTORY_ACTION_TYPES, mergeWithDetail }) => {
+      store.replaceReducer(
+        updateReducers({
+          ...mergeWithEntities(systemReducer([], INVENTORY_ACTION_TYPES), {
+            page: Number(filters.offset / filters.limit + 1 || 1),
+            perPage: Number(filters.limit || 20),
+          }),
+          ...mergeWithDetail(),
+        })
+      );
+    },
+    [filters, store]
+  );
+};
+
+export const mergeAppColumns = (defaultColumns) => {
+  const lastSeenColumn = defaultColumns.find(({ key }) => key === 'updated');
+  const impacted_date = {
+    key: 'impacted_date',
+    title: 'First Impacted',
+    sortKey: 'impacted_date',
+    transforms: [sortable, wrappable],
+    props: { width: 15 },
```

Comment:
I am not sure this is the standard way we use to change the columns width. Shouldn't `cellWidth` transform property serve you better? https://www.patternfly.org/2022.11/components/table/react-legacy#controlling-text

Reply:
As you can see these columns are passed to the inventory table in the middle. InventoTable then it gets mapped to cellWith down the line here: https://github.com/RedHatInsights/insights-inventory-frontend/blob/e9af8e577b5c088d4f377b11218393e7918ca9af/src/components/InventoryTable/helpers.js#L120